### PR TITLE
chore(shorebird_cli): use currentPeriodCostMoney in account usage command

### PR DESCRIPTION
**Pull request by bryanoltman**
## Description

Replaces manual cents-to-dollars conversion with a `Money` object's `toString()`

Fixes https://github.com/shorebirdtech/shorebird/issues/803

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

### CODE REVIEW

File: packages/shorebird_cli/lib/src/commands/account/account_usage_command.dart [hunk 1]
```diff
@@ -99,7 +97,7 @@ ${Table(
 ${styleBold.wrap('${lightCyan.wrap(remainingPatchInstalls)} patch installs remaining in the current billing period.')}
 
 Current Billing Period: ${lightCyan.wrap(DateFormat.yMMMd().format(currentPeriodStart))} - ${lightCyan.wrap(DateFormat.yMMMd().format(currentPeriodEnd))}
-Month-to-date cost: ${lightCyan.wrap('\$${currencyFormatter.format(currentPeriodCost / 100.0)}')}
+Month-to-date cost: ${lightCyan.wrap(currentPeriodCostMoney.toString())}
```
Comment (eseidel) [inline packages/shorebird_cli/lib/src/commands/account/account_usage_command.dart]: Does currentPeriodCost get deleted now?

**Review by eseidel — APPROVED** [review]
🎶 ode to joy 🎶

File: packages/shorebird_cli/lib/src/commands/account/account_usage_command.dart (hunk 1 above)
Comment (bryanoltman) [reply packages/shorebird_cli/lib/src/commands/account/account_usage_command.dart]: Unfortunately no – the `Money` type doesn't serialize to json, so we'll need to transport this value over the wire as an int.

**Comment by eseidel** [comment]
We may also end up just writing our own Currency type / wrapping this one.  🤷

File: packages/shorebird_cli/lib/src/commands/account/account_usage_command.dart (hunk 1 above)
Comment (bryanoltman) [reply packages/shorebird_cli/lib/src/commands/account/account_usage_command.dart]: nvm, updated in the base branch :)

**Comment by bryanoltman** [comment]
Merging this into the base branch (bo/money) as the tests won't pass there without these changes.

**Pull request merged**